src/components/illustrationList/illustrationList.module.css

```
@@ -0,0 +1,23 @@
+:not(.illustrationlist) + .illustrationList {
```

Ah. Nå ser jeg det. Feil casing
```suggestion
:not(.illustrationList) + .illustrationList {
```